integration/use-date-string/use-date-string.ts

```diff
@@ -0,0 +1,253 @@
+/* eslint-disable */
+import { Writer, Reader } from 'protobufjs/minimal';
+
+export const protobufPackage = '';
+
+export interface Todo {
+  id: string;
+  timestamp: string | undefined;
+  repeatedTimestamp: string[];
+  optionalTimestamp?: string | undefined;
+  mapOfTimestamps: { [key: string]: string };
+}
+
+export interface Todo_MapOfTimestampsEntry {
+  key: string;
+  value: string | undefined;
+}
+
+const baseTodo: object = { id: '', timestamp: '', repeatedTimestamp: '' };
+
+export const Todo = {
+  encode(message: Todo, writer: Writer = Writer.create()): Writer {
+    if (message.id !== '') {
+      writer.uint32(10).string(message.id);
+    }
+    if (message.timestamp !== undefined) {
+      writer.uint32(18).string(message.timestamp);
+    }
+    for (const v of message.repeatedTimestamp) {
+      writer.uint32(26).string(v!);
+    }
+    if (message.optionalTimestamp !== undefined) {
+      writer.uint32(34).string(message.optionalTimestamp);
+    }
+    Object.entries(message.mapOfTimestamps).forEach(([key, value]) => {
+      Todo_MapOfTimestampsEntry.encode({ key: key as any, value }, writer.uint32(42).fork()).ldelim();
+    });
+    return writer;
+  },
+
+  decode(input: Reader | Uint8Array, length?: number): Todo {
+    const reader = input instanceof Uint8Array ? new Reader(input) : input;
+    let end = length === undefined ? reader.len : reader.pos + length;
+    const message = globalThis.Object.create(baseTodo) as Todo;
+    message.repeatedTimestamp = [];
+    message.mapOfTimestamps = {};
+    while (reader.pos < end) {
+      const tag = reader.uint32();
+      switch (tag >>> 3) {
+        case 1:
+          message.id = reader.string();
+          break;
+        case 2:
+          message.timestamp = reader.string();
```

Comment:
I think this should decode the timestamp using Timestamp.decode(reader, reader.uint32()) and then convert it to string.